src/pages/[platform]/build-a-backend/storage/copy-files/index.mdx

```diff
@@ -114,6 +114,9 @@ Option | Type | Default | Description |
 | -- | :--: | :--: | ----------- |
 | path | string \| <br/>(\{ identityId \}) => string | Required | A string or callback that represents the path in source and destination bucket to copy the object to or from. <br /> **Each segment of the path in `source` must by URI encoded.** |
 | bucket | string \| <br />\{ bucketName: string;<br/> region: string; \} | Default bucket and region from Amplify configuration | A string representing the target bucket's assigned name in Amplify Backend or an object specifying the bucket name and region from the console.<br/><br/>Read more at [Configure additional storage buckets](/[platform]/build-a-backend/storage/set-up-storage/#configure-additional-storage-buckets). |
+| eTag | string | Optional | The copy **source object** entity tag (ETag) value. Only Copies the object if its ETag matches the specified tag. |
+| notModifiedSince | Date | Optional | Copies the **source object** if it hasn't been modified since the specified time.  <br /><br/> **This is evaluated only when `eTag` is NOT supplied**|
+| expectedBucketOwner | string | Optional | The account ID that owns source or destination bucket. |
```

Comment:
The account ID passed must own both source and destination buckets?

Reply:
There's an `expectedBucketOwner` option for each of `source` and `destination` parameter, and it's used for source and destination bucket respectively. Maybe I need to make the wording clearer:

```suggestion
| expectedBucketOwner | string | Optional | `source.expectedBucketOwner`: The account ID that owns the source bucket. <br /><br /> `destination.expectedBucketOwner`: The account ID that owns the destination bucket |
```